public/app/features/dimensions/editors/ScalarDimensionEditor.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FC, useCallback } from 'react';
+import { FieldType, GrafanaTheme2, SelectableValue, StandardEditorProps } from '@grafana/data';
+import { ScalarDimensionConfig, ScalarDimensionMode, ScalarDimensionOptions } from '../types';
+import { InlineField, InlineFieldRow, RadioButtonGroup, Select, useStyles2 } from '@grafana/ui';
+import { useFieldDisplayNames, useSelectOptions } from '@grafana/ui/src/components/MatchersUI/utils';
+import { NumberInput } from './NumberInput';
+import { css } from '@emotion/css';
+
+const fixedValueOption: SelectableValue<string> = {
+  label: 'Fixed value',
+  value: '_____fixed_____',
+};
+
+const scalarOptions = [
+  { label: 'Mod', value: ScalarDimensionMode.Mod, description: 'Use values, mod from max' },
```

Comment:
May want to specify `Use field values` similar to what we have below just to be explicit